packages/bitcore-wallet-service/src/lib/chain/eth/index.ts

```diff
@@ -215,6 +215,7 @@ export class EthChain implements IChain {
               data,
               gasPrice
             });
+            gasLimit = gasLimit + gasLimit * Defaults.MS_GAS_LIMIT_BUFFER_PERCENT;
```

Comment:
Need to convert this to an int

```
FpNUAGe6RUZ79o3FXeJeXB: Error: Error: [number-to-bn] while converting number 148679.3 to BN.js instance, error: invalid number value. Value must be an integer, hex string, BN or BigNumber instance. Note, decimals are not supported. Given value: "148679.3"
```

This can also be cleaned up a little to 
```
gasLimit += gasLimit * Defaults.MS_GAS_LIMIT_BUFFER_PERCENT;
```